index.d.ts

```diff
@@ -197,10 +204,10 @@ export declare class API {
 
 
 
-  use(path: string, ...middleware: Middleware[]);
-  use(paths: string[], ...middleware: Middleware[]);
-  use (...middleware: Middleware[]);
-  use (...errorHandlingMiddleware: ErrorHandlingMiddleware[]);
+  use(path: string, ...middleware: Middleware[]): void;
+  use(paths: string[], ...middleware: Middleware[]): void;
+  use(...middleware: Middleware[]): void;
```

Comment:
I think you can improve this even further by allowing mixing normal middleware with error-handling middleware, cause if I see correctly the library allows it. Something like this, but prettier of course ;P :
```typescript
type AnyMiddleware = Middleware | ErrorHandlingMiddleware

use(path: string, ...middleware: Middleware[]): void;
use(paths: string[], ...middleware: Middleware[]): void;
use(...middleware: AnyMiddleware[]): void;
```